gui-apps/foot/foot-1.10.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson xdg
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="https://codeberg.org/dnkl/foot/archive/${PV}.tar.gz  -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://codeberg.org/dnkl/foot.git"
+fi
+
+DESCRIPTION="A fast, lightweight and minimalistic Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+LICENSE="MIT"
+SLOT="0"
+IUSE="ime +themes +grapheme-clustering"
+
+DEPEND="
+	dev-libs/libutf8proc:=
+	dev-libs/wayland
```

Comment:
Haven't tried it but packages depending on wayland typically need wayland-scanner in bdepend.

/usually/ you'll have it, but wayland-scanner can be depcleaned or missing esp if mixing some binpkgs.